MovieApp/MovieApp/App/Features/MovieBrowser/Components/MovieCollectionViewCell/FavouriteButton.swift

```diff
@@ -0,0 +1,23 @@
+import UIKit
+
+class FavouriteButton: UIButton {
+    
+    let size: CGFloat = 32
+        
+    init() {
+        let rect = CGRect(x: 0, y: 0, width: size, height: size)
+        super.init(frame: rect)
```

Comment:
We try to avoid using `init(.frame: sth)`, rather use `frame: .zero` and set the size out of that button.
This is a general rule for creating components. They can have the expected size (like you added `let size: CGFloat = 32`) but outside component should be responsible for laying out its subviews, and the subviews should be able to draw themselves anywhere.

Reply:
I [used](https://github.com/lihter/Movie-App/pull/24/files#diff-88c1fbecca5f8c63bb56759d56940d4d5ba24af98bcfc34249c92a7ef5bb2034R8) `.zero`.